File: TECAir_API/Models/Usuario.cs
namespace TECAir_API.Models
{
    public class Usuario
    {
        public int id_usuario { get; set; }
        public string u_nombre { get; set; }
        public string u_apellido1 { get; set; }
        public string u_apellido2 { get; set; }
        public string correo { get; set; }
        public string u_contrasena { get; set; }
        public int telefono  { get; set; }

        public Usuario()
        {
        }

        public Usuario(int id, string nombre, string apellido1, string apellido2, string correo, string contrasena, int telefono)
        {
            this.id_usuario = id;
            this.u_nombre = nombre;
            this.u_apellido1 = apellido1;
            this.u_apellido2 = apellido2;
            this.correo = correo;
            this.u_contrasena = contrasena;
            this.telefono = telefono;
        }
    }
}

File: TECAir_APPMOVIL/app/src/main/java/com/example/tecair/db/entities/Usuario.java
package com.example.tecair.db.entities;

public class Usuario {
    private int id_usuario;
    private String u_nombre;
    private String u_apellido1;
    private String u_apellido2;
    private String correo;
    private String u_contrasena;
    private int telefono;

    public int getId_usuario() {
        return id_usuario;
    }

    public void setId_usuario(int id_usuario) {
        this.id_usuario = id_usuario;
    }

    public String getU_nombre() {
        return u_nombre;
    }

    public void setU_nombre(String u_nombre) {
        this.u_nombre = u_nombre;
    }

    public String getU_apellido1() {
        return u_apellido1;
    }

    public void setU_apellido1(String u_apellido1) {
        this.u_apellido1 = u_apellido1;
    }

    public String getU_apellido2() {
        return u_apellido2;
    }

    public void setU_apellido2(String u_apellido2) {
        this.u_apellido2 = u_apellido2;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getU_contrasena() {
        return u_contrasena;
    }

    public void setU_contrasena(String u_contrasena) {
        this.u_contrasena = u_contrasena;
    }

    public int getTelefono() {
        return telefono;
    }

    public void setTelefono(int telefono) {
        this.telefono = telefono;
    }
}
